drift_dev/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: drift_dev
 description: Dev-dependency for users of drift. Contains a the generator and development tools.
-version: 2.3.0
+version: 2.4.0-dev
```

Comment:
Please also change the `drift: ` dependency in this package to `>=2.4.0 <2.5.0`.